package.json

```diff
@@ -1,38 +1,45 @@
 {
-    "name": "interplanetaryfonts",
-    "homepage": "https://github.com/interplanetaryfonts/interplanetaryfonts.xyz#readme",
-    "version": "0.1.0",
-    "private": true,
-    "description": "InterplanetaryFonts Fullstack DApp",
-    "main": "README.md",
-    "scripts": {
-        "compile:smart-contract": "pnpm --filter contracts run compile",
-        "deploy:smart-contract": "pnpm --filter contracts run deploy",
-        "upgrade:smart-contract": "pnpm --filter contracts run upgrade",
-        "verify:smart-contract": "pnpm --filter contracts run verify",
-        "integration-test:smart-contract": "pnpm --filter contracts run integration-test",
-        "test:smart-contract": "pnpm --filter contracts run test",
-        "dev:frontend": "pnpm --filter frontend run dev",
-        "dev:backend": "pnpm --filter backend run dev"
-    },
-    "repository": {
-        "type": "git",
-        "url": "git+https://github.com/interplanetaryfonts/interplanetaryfonts.xyz.git"
-    },
-    "keywords": [
-        "ethereum",
-        "blockchain",
-        "web3",
-        "polygon",
-        "graphQL",
-        "superfluid",
-        "fonts",
-        "fullstack"
-    ],
-    "author": "InterplanetaryFonts",
-    "license": "MIT",
-    "dependencies": {
-        "graphql": "^16.6.0",
-        "urql": "^3.0.3"
-    }
+  "name": "interplanetaryfonts",
+  "homepage": "https://github.com/interplanetaryfonts/interplanetaryfonts.xyz#readme",
+  "version": "0.1.0",
+  "private": true,
+  "description": "InterplanetaryFonts Fullstack DApp",
+  "main": "README.md",
+  "scripts": {
+    "compile:smart-contract": "pnpm --filter contracts run compile",
+    "deploy:smart-contract": "pnpm --filter contracts run deploy",
+    "upgrade:smart-contract": "pnpm --filter contracts run upgrade",
+    "verify:smart-contract": "pnpm --filter contracts run verify",
+    "integration-test:smart-contract": "pnpm --filter contracts run integration-test",
+    "test:smart-contract": "pnpm --filter contracts run test",
+    "dev:frontend": "pnpm --filter frontend run dev",
+    "lint:frontend": "pnpm --filter frontend run lint",
+    "prettier:frontend": "pnpm --filter frontend run prettier",
```

Comment:
Do we now need to run these 2 scripts before pushing new changes JP?

Reply:
I setup a pre commit hook that will run lint-staged commands on the changed files before a commit is created...
https://github.com/interplanetaryfonts/interplanetaryfonts.xyz/blob/008281b6c02bcfc2ba686e44290cf0b206a4c2bb/.husky/pre-commit#L1-L4

This lint staged config file declares that all changed js and jsx files will be run through eslint and that all changed files get ran through prettier...
https://github.com/interplanetaryfonts/interplanetaryfonts.xyz/blob/008281b6c02bcfc2ba686e44290cf0b206a4c2bb/frontend/.lintstagedrc.js#L1-L12

lint-staged repo ( Run linters on git staged files ) https://github.com/okonet/lint-staged
husky docs ( makes creating git hooks easy ) https://typicode.github.io/husky/#/